specs/integration/search/CreateReuseWidget.js

```diff
@@ -0,0 +1,18 @@
+contributeLink: 'a[title="Contribute!"]',
+publishReuseLink: ".actions-list a:first-child + a",
+nextLink: "button.pointer:last-child",
+titleField: "input#title",
+urlField: "input#url",
+descriptionField: "textarea#description",
+setTypeOption: 'select#type option[value="post"]',
+openFileUploaderLink: '.image-picker input[type="file"]',
+
+fill: function fill(name, url, description) {
```

Comment:
I hesitate between the two options as described [in the documentation](https://github.com/MattiSG/Watai/wiki/Features#example), I'm wondering if I left the responsibility to fill the whole form(s) to widgets or features.